metastats-domain/src/main/groovy/life/qbic/metastats/filter/SequencingModeCalculator.groovy

```diff
@@ -14,33 +12,31 @@ import org.apache.logging.log4j.Logger
  */
 class SequencingModeCalculator {
 
-    private static final Logger LOG = LogManager.getLogger(SequencingModeCalculator.class)
-
     /**
      * Calculates the sequencing mode based on all filenames of a sample.
      * If the files contain R1 and R2 then the mode is paired end. Else it is single end.
```

Comment:
You have an enum here.

Reply:
You mean I should state that in the description here?